src/meson.build

```diff
@@ -7,17 +7,29 @@ screensaver_command_sources = [
     'gnome-screensaver-command.c',
 ]
 
+
+if host_machine.system() in ['openbsd', 'freebsd', 'netbsd']
```

Comment:
Not sure why this block is here if the sources are set to the same thing regardless. Should remove this change entirely.